### ISSUE TITLE
requires_gradients_ is being called on model_args, which are not guaranteed to be tensors nor be connected via gradients

### ISSUE THREAD

**Issue by rshih32** [truera/trulens#61]
in models.pytorch 
```
model_args = ModelWrapper._nested_apply(
                model_args, lambda model_args: model_args.requires_grad_(True))
```

**Comment by piotrm0** [comment]
Fixed by https://github.com/truera/trulens/pull/58 .
